tasks/gulp/compile-assets.js

```diff
@@ -54,42 +64,6 @@ gulp.task('scss:compile', () => {
       })
     ))
     .pipe(gulp.dest(taskArguments.destination + '/css/'))
-
-  let compileOldIe = gulp.src(configPaths.app + 'assets/scss/govuk-frontend-old-ie.scss')
```

Comment:
I don't understand why this is being removed as part of this PR. Don't we still need an old-ie stylesheet?

Reply:
we still have old ie stylesheet. oldie was not doing anything that we don't currently do at the momene (flatten mq's use rems)

Reply:
Can that change be made in a separate PR? It doesn't seem related to the other changes.

(I think there are some things oldie might be doing or should be doing – patching `::before` into `:before` (which is currently disabled??), changing opacity to filters – but I don't want to hold this PR up)

Reply:
ok, can leave it in and will add required merge-stream back in